daemon/daemon_no_aufs.go

```diff
@@ -1,4 +1,4 @@
-// +build exclude_graphdriver_aufs,linux
+// +build exclude_graphdriver_aufs,linux !linux
```

Comment:
didnt know aufs exist on windows, is it ok to rewrite it as // +build exclude_graphdriver_aufs freebsd ?